### DISCUSSION TITLE
Error in post-command-hook (vertico--exhibit): (void-function marginalia--orig-completion-metadata-get

### DISCUSSION THREAD

**Discussion by zenny** [minad/vertico#505] [Q&A] [answered]
Hi,

I am using https://github.com/pprevos/emacs-writing-studio, but a recent upgrade spits out an error that disables the `vertico` panel to appear.

```
Error in post-command-hook (vertico--exhibit): (void-function marginalia--orig-completion-metadata-get
```

I removed the `vertico` package and reinstalled again as advised in  https://github.com/minad/vertico/discussions/50, but no go.

Cheers,
/z

**Comment by erpreciso** [comment, marked as answer]
From `list-packages` I removed then reinstalled `vertico`, then removed then reinstalled `marginalia`, and it solved the issue.
Version: `GNU Emacs 30.0.50`

**Reply by zenny** [reply]
> From `list-packages` I removed then reinstalled `vertico`, then removed then reinstalled `marginalia`, and it solved the issue. Version: `GNU Emacs 30.0.50`

Thanks @erpreciso

**Reply by sevillaarvin** [reply]
> From `list-packages` I removed then reinstalled `vertico`, then removed then reinstalled `marginalia`, and it solved the issue. Version: `GNU Emacs 30.0.50`

This worked! `M-x package-reinstall` `vertico` and `M-x package-reinstall` `marginalia`
